GVFS/GVFS.Installer.Mac/scripts/preinstall

```diff
@@ -4,3 +4,12 @@ if [ -f /Library/LaunchDaemons/org.vfsforgit.prjfs.PrjFSKextLogDaemon.plist ]; t
     echo "Unloading PrjFSKextLogDaemon: '$unloadCmd'..."
     eval $unloadCmd || exit 1
 fi
+
+KEXTBUNDLEID="org.vfsforgit.PrjFSKext"
+isKextLoadedCmd="/usr/sbin/kextstat -l -b $KEXTBUNDLEID | wc -l"
+isKextLoaded=$(eval $isKextLoadedCmd)
+if [ "$isKextLoaded" -gt 0 ]; then
+	unloadCmd="sudo /sbin/kextunload -b $KEXTBUNDLEID"
```

Comment:
Where will the prompt for the sudo password be when this is ran?

Reply:
Installer only runs elevated - so it wouldn't need to display the prompt.

Reply:
Why bother with `sudo` at all?

Reply:
Also, we should probably check for the legacy kext identifier here too, at least for now?

Reply:
Since we have so few early adopter customers, I'd rather tell them just to fully uninstall VFS4G and then reinstall before shipping our next release rather than bake in the backwards compatibility.

Thoughts?
cc: @wilbaker @jrbriggs

Reply:
Removed `sudo`.

Reply:
> I'd rather tell them just to fully uninstall VFS4G and then reinstall

If we are going to hold the next release to EA for this PR then I agree with Phil's suggestion.  All the EA users currently have the legacy kext identifier, and so for these changes to work correctly for those users we'll need to check the legacy kext identifier.  

I agree we don't need this long term though, and so we could create a work item (or even a PR) to remove it for the next release.

cc: @jrbriggs for his thoughts

Reply:
+1 for Phil's suggestion and I agree with the follow-up work to remove it.

Reply:
Added the check for legacy kext as well. Created [work item](https://github.com/Microsoft/VFSForGit/issues/891) to remove this check in future.